src/AppETH.tsx

```diff
@@ -52,6 +56,19 @@ function AppETH ({ contract, defaultAccount, web3 }: Props): React.ReactElement<
   const [polkadotRecipient, setPolkadotRecipient] = useState(String);
   const [depositAmount, setDepositAmount] = useState(String);
 
+  const [polkaEthBalance, setPolkaEthBalance] = useState(String);
+
+  if(defaultAccount === undefined) {
+    alert("Please install MetaMask to use this application!");
+    return(<div><p>Please install MetaMask to use this application!</p></div>)
```

Comment:
we should probably have a button here to try connect to metamask again if first time fails or something